catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -104,8 +108,17 @@ public UpdateRequest process(UpdateRequest input)
                 validator.validate(metacard);
             } catch (ValidationException e) {
                 String validatorName = getValidatorName(validator);
+                boolean validationErrorsExist = CollectionUtils.isNotEmpty(e.getErrors());
+                boolean validationWarningsExist = CollectionUtils.isNotEmpty(e.getWarnings());
 
-                if (isValidatorEnforced(validatorName)) {
+                if (isValidatorEnforced(validatorName) && validationErrorsExist && enforceErrors) {
+                    INGEST_LOGGER.info(
```

Comment:
Is `info` really the right level here? I would think it should either be `warn` or `debug` depending on how important/common we think this will be.